packages/backend/src/utils.ts

```diff
@@ -59,27 +59,32 @@ export function toText(e: Error): string {
   return _.get(e, "message") || _.get(e, "name", _.toString(e));
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function getEnvResources(envFilePath: string): Promise<any> {
+export function getEnvResources(envFilePath: string): Promise<{ vcapObject: any; isQuotedVcap: boolean }> {
   try {
+    let isQuotedVcap = false;
     if (existsSync(envFilePath)) {
       const envProperties = PropertiesReader(envFilePath);
-      const vcapProperty = envProperties.getRaw(ENV_VCAP_RESOURCES);
+      let vcapProperty = envProperties.getRaw(ENV_VCAP_RESOURCES);
```

Comment:
please revert it to 'const'

Reply:
it cannot be const - I may do afterwards new assignment